mkdocs.yml

```diff
@@ -2,18 +2,21 @@ site_name: dynamiqs
 site_url: "https://www.dynamiqs.org"
 use_directory_urls: false
 repo_url: https://github.com/dynamiqs/dynamiqs
+repo_name: dynamiqs
 theme:
   name: material
+  custom_dir: docs/overrides
   features:
     - navigation.instant
     - navigation.tabs
     - navigation.sections
     - navigation.indexes
     - navigation.top
+    - navigation.path
```

Comment:
You can remove this, it's for sponsors only (people paying for the dvpmt of mkdocs-material) > https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-path.